volunteers.js

```diff
@@ -347,3 +347,16 @@ async function getCurrentMilestone(googleClient) {
 
   return bestMilestone;
 }
+
+async function discordIdToTag(guild, id) {
+    const match = id.match(/<@!?(\d+)>/);
+    if (match !== null) {
+        try {
+            volunteerMember = await guild.fetchMember(match[1]);
```

Comment:
I haven't tried it, but does `fetchMember` work with string ids?

https://discord.js.org/#/docs/main/11.4.2/class/Guild?scrollTo=fetchMember

Reply:
Yep, cause an ID is always a string in JavaScript because js doesn't play nice with 64bit numbers (which is what the ID is)

Reply:
I see, cool.